src/bmap.ml

```diff
@@ -23,47 +29,39 @@ module type S = sig
       [k] is removed from [m], and if the result is [Some v'] then
       [add k v' m] is returned. *)
 
-  val merge_safe :
+  val mergeSafe :
     f:(key -> [`Left of 'a | `Right of 'b | `Both of 'a * 'b] -> 'c option) ->
     'a t -> 'b t -> 'c t
-  (** [merge_safe ~f a b] merges the maps [a] and [b] together.
+  (** [mergeSafe ~f a b] merges the maps [a] and [b] together.
       @since 0.17 *)
 
-  val of_seq : (key * 'a) sequence -> 'a t
+  val ofList : (key * 'a) list -> 'a t
 
-  val add_seq : 'a t -> (key * 'a) sequence -> 'a t
+  val addList : 'a t -> (key * 'a) list -> 'a t
   (** @since 0.14 *)
 
-  val to_seq : 'a t -> (key * 'a) sequence
```

Comment:
I've kept `toSeq` in Option (there was no `ofSeq`) because it seems like a handy way of converting between different containers. I've dropped everything else sequence-related though. Is there a reason to remove these, other than just to simplify the interface?